.github/workflows/e2e-consuming.yaml

```
@@ -13,7 +13,7 @@ jobs:
     strategy:
       fail-fast: false
       matrix:
-        project: ['submariner', 'submariner-operator', 'lighthouse']
+        project: ['admiral', 'submariner', 'submariner-operator', 'lighthouse']
         deploytool: ['operator', 'helm']
         exclude:
```

```suggestion
        exclude:
          - project: admiral
            deploytool: helm
```

We should exclude helm as it's redundant in the case of admiral and admiral repo doesnt test it anyhow

---

Done!